lib/ts/recipe/emailpassword/recipe.ts

```diff
@@ -101,6 +105,63 @@ export default class Recipe extends RecipeModule {
                 Recipe.instance = new Recipe(Recipe.RECIPE_ID, appInfo, isInServerlessEnv, config, {
                     emailDelivery: undefined,
                 });
+
+                PostSuperTokensInitCallbacks.addPostInitCallback(() => {
+                    const mfaInstance = MultiFactorAuthRecipe.getInstance();
+                    if (mfaInstance !== undefined) {
+                        mfaInstance.addGetAllFactorsFromOtherRecipesFunc((tenantConfig) => {
+                            if (tenantConfig.emailPassword.enabled === false) {
+                                return [];
+                            }
+                            return ["emailpassword"];
+                        });
+                        mfaInstance.addGetFactorsSetupForUserFromOtherRecipes(async (user: User) => {
```

Comment:
In emailpassword, we have to consider the tenantId as well, otherwise the user may end up calling the sign in API with an email that belongs to another tenant.

Reply:
done.